ui/src/components/about/StaffSection.js

```diff
@@ -75,43 +75,6 @@ const StaffSection = (props) => {
             </Box>
           </Stack>
         </Box>
-        <Box bgColor="#2D4769" w="100%" borderRadius="20px">
-          <Stack
-            direction={{ base: 'column', lg: 'row' }}
-            py={['2em', '2em', '4em']}
-          >
-            <Image
-              src={process.env.PUBLIC_URL + '/images/about/ZhuFamily.png'}
```

Comment:
should we remove these assets that we don't use anymore? so it doesn't take up space

Reply:
removed